tester/tester.go

```diff
@@ -193,9 +194,18 @@ func validateHTTPCode(contract Contract, resp *http.Response) error {
 
 func validateResponseBody(contract Contract, body []byte) error {
 	if contract.ExpectedResponseBody != "" {
-		if !strings.Contains(string(body), contract.ExpectedResponseBody) {
-			return fmt.Errorf("expected response not found in the body")
+		if !strings.HasPrefix(contract.ExpectedResponseBody, "r/") {
+			if !strings.Contains(string(body), contract.ExpectedResponseBody) {
+				return fmt.Errorf("expected response not found in the body")
+			}
+		} else {
+			expectedRegexp := contract.ExpectedResponseBody[2:]
+			re := regexp.MustCompile(expectedRegexp)
```

Comment:
`MustCompile` panics if the regexp doesn't compile.  What if we used `Compile` and treated any compilation errors as a sign that the user did not pass a regexp, but rather an expected body that starts with r/ ?   What do you think?

Reply:
Oh sorry, dude, I thought I was using Compile, it's fixed (thanks for the review)
I have also implemented your great idea, to consider it as a normal string (there are a test on the smoke_test file, to test that as well)